src/components/CentralContainer/Sidebar/SidebarTabs/CustomTabPanel.css

```diff
@@ -2,12 +2,19 @@
     padding: 0.75rem;
 }
 
+.custom-tab-panel-container {
```

Comment:
For consistency, shall we name this as `sidebar-tab-panel-container` or update the other classes' names?